app/routes/public/sessions.js

```diff
@@ -146,35 +146,10 @@ export default class SessionsRoute extends Route {
   }
 
   async getSchedules(event) {
-    const scheduledFilterOptions = [
-      {
-        and: [
-          {
-            name : 'starts-at',
-            op   : 'ne',
-            val  : null
-          },
-          {
-            or: [
-              {
-                name : 'state',
-                op   : 'eq',
-                val  : 'accepted'
-              },
-              {
-                name : 'state',
-                op   : 'eq',
-                val  : 'confirmed'
-              }
-            ]
-          }
-        ]
-      }
-    ];
 
     const sessions = await event.query('sessions', {
-      filter       : scheduledFilterOptions,
-      'page[size]' : 0
+      'fields[session]' : 'starts-at',
+      'page[size]'      : 0
     });
```

Comment:
Not this includes non accepted sessions

Reply:
ok then i should add accepted and confirmed sessions